dev/acf-export.json

```diff
@@ -4392,62 +4337,6 @@
                 "preview_size": "header-img-md",
                 "library": "all"
             },
-            {
-                "key": "field_58f7a778185ef",
-                "label": "Header Image (-sm+)",
-                "name": "page_header_image",
-                "aria-label": "",
-                "type": "image",
-                "instructions": "Header image to display at the -sm (768px width) breakpoint and up.",
-                "required": 0,
-                "conditional_logic": 0,
-                "wrapper": {
-                    "width": "",
-                    "class": "",
-                    "id": ""
-                },
-                "relevanssi_exclude": 0,
-                "uploader": "",
-                "return_format": "id",
-                "acfe_thumbnail": 0,
-                "min_width": 768,
-                "min_height": "",
-                "min_size": "",
-                "max_width": "",
-                "max_height": "",
-                "max_size": "",
-                "mime_types": "png,jpg,jpeg",
-                "preview_size": "header-img-sm",
-                "library": "all"
-            },
-            {
-                "key": "field_58f7a7b8185f0",
-                "label": "Header Image (-xs+)",
-                "name": "page_header_image_xs",
-                "aria-label": "",
-                "type": "image",
-                "instructions": "Header image to display at the -xs (576px width) breakpoint and up.",
-                "required": 0,
-                "conditional_logic": 0,
-                "wrapper": {
-                    "width": "",
-                    "class": "",
-                    "id": ""
-                },
-                "relevanssi_exclude": 0,
-                "uploader": "",
-                "return_format": "id",
-                "acfe_thumbnail": 0,
-                "min_width": 576,
-                "min_height": "",
-                "min_size": "",
-                "max_width": "",
-                "max_height": "",
-                "max_size": "",
-                "mime_types": "png,jpg,jpeg",
-                "preview_size": "header-img",
-                "library": "all"
-            },
```

Comment:
Did you intend to remove both of these? It looks like we're back down to 2 sizes, `lg` and `md`. Did we mean to keep `xs`?

Reply:
It was removed accidentally. Added the XS size.